CHANGELOG.rst

```diff
@@ -1,5 +1,45 @@
 Changelog
 =========
+1.14.0 (2021-02-18)
+-------------------
+MLflow 1.14.0 includes several major features and improvements:
+
+Support for tensor inputs using the TF serving format:
```

Comment:
I think we could actually describe this in a single line item, but link to all the relevant PRs. 

```suggestion
- MLflow's model inference APIs  (`mlflow.pyfunc.predict`), built-in model serving tools (`mlflow models serve`), and model signatures now support tensor inputs. In particular, MLflow's integrations with PyTorch, TensorFlow, Keras, ONNX, and Gluon support tensor input output of the box. For more information, see https://mlflow.org/docs/latest/models.html#deploy-mlflow-models (#3894 @wentinghu ...)
```